scripts/globals/mobskills/arctic_impact.lua

```diff
@@ -0,0 +1,26 @@
+---------------------------------------------------
+-- Arctic Impact
+-- Deals Ice damage to enemies within range.
+-- Area of Effect is centered around caster.
+---------------------------------------------------
+require("scripts/settings/main")
```

Comment:
It doesn't look like any settings are used in this file, so this require isn't needed.